docs/application/tizen-studio/setup/prerequisites.md

```diff
@@ -3,8 +3,11 @@
 Check the following prerequisites before attempting to install Tizen Studio.
 
 ## Java Development Kit (JDK) Requirements
+> **Note**
+>
+> There are no JDK prerequisites for Tizen Studio version 3.7 and higher.
```

Comment:
@mijin85cho,

In my opinion the suggested sentence "There are no JDK prerequisites for Tizen Studio version 3.7 and higher." is more direct as compared to Since Tizen Studio version 3.7 and higher, JDK prerequisites are not required.
The second suggestion is also ambiguous, for example, do the JDK prerequisites exist and why its not required.
Please let me know your views.

Reply:
Regarding the second opinion : The intent was to say the JDK prerequisites exists for the previous version, but it is not required for Tizen Studio 3.7 (and higher versions)

Anyway, I think current sentence is also acceptable so will keep it if you want.

Just one another suggestion to it : how about removing the _version_ after _Tizen Studio_?
In the Tizen Studio pages as you can see we usually don't use "version" after Tizen Studio.

Reply:
@mijin85cho,

As per google style guid: https://developers.google.com/style/word-list 
The usage is version before the version number.

Reply:
I think the point of the guideline is on using "higher" instead of "later" or "+",
not on using "version".

Also using "version" after "Tizen Studio" doesn't match with[ Tizen document style](https://github.com/Samsung/tizen-docs/blob/master/styleguide/naming-rules.md#how-to-use-proper-nouns),
as you can see in the pages including https://docs.tizen.org/application/tizen-studio/index.

Even more is, the next line (line 10) we are using "Tizen Studio 3.5 and Tizen Studio 3.6".

Reply:
@mijin85cho, thank you for the explanation, will remove version.

@adityaaswani, request you to please update the sentence as follows:

There are no JDK prerequisites for Tizen Studio 3.7 and higher.